src/promptflow-evals/samples/built_in_evaluators.py

```diff
@@ -114,7 +115,7 @@ def run_safety_evaluators():
 
 
 def run_qa_evaluator():
-    qa_eval = QAEvaluator(model_config=model_config, deployment_name="gpt-4")
+    qa_eval = QAEvaluator(model_config=model_config, deployment_name=deployment_name)

```

Comment:
I sort of remember that Clement mentioned below needs to be changed as well.

from promptflow.entities import AzureOpenAIConnection

Reply:
I meant to say this change is missed for unit-test files